src/media/js/elements/nav.js

```diff
@@ -151,6 +151,13 @@ define('elements/nav',
                         if (this.isLeafPage) {
                             logger.log('hambacker button pressed (back)');
                             navigation.back();
+                        } else if (this.isCategoryPage) {
```

Comment:
Yeah, we need a component that does this which makes me think it should be a `<mkt-hambacker>`. Should be easy enough to extract this later if we add this menu to something else.